### ISSUE TITLE
FATAL response after initial startup

### ISSUE THREAD

**Issue by vbaidak** [cloudfoundry/uaa#492]
# Steps

1. $ git clone git://github.com/cloudfoundry/uaa.git
2. $ cd uaa
3. $ ./gradlew run

# Expected

UAA is up with no errors && endpoints available

# Actual

*localhost:8080* and any other with paths like /uaa give *FATAL* response

Log output:

```txt
[2016-11-27 13:14:01.092] cloudfoundry-identity-server/uaa - ???? [localhost-startStop-1] .... FATAL --- RecognizeFailureDispatcherServlet: org.springframework.beans.factory.BeanDefinitionStoreException: Invalid bean definition with name 'identityZoneConfigurationBootstrap' defined in ServletContext resource [/WEB-INF/spring-servlet.xml]: Could not resolve placeholder 'login.serviceProviderKey' in string value "${login.serviceProviderKey}"; nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'login.serviceProviderKey' in string value "${login.serviceProviderKey}"
```

**Comment by vbaidak** [comment]
Seems that current version in master and 3.9.2 release version is not working.
Version 3.9.1 works normally.

Please have a look
Thanks

**Comment by sreetummidi** [comment]
Addressed in 3.16.0
